### PR TITLE
Conflict Blender with Blender LTS

### PR DESCRIPTION
**Pull request by walles**
As requested here:
<https://github.com/Homebrew/homebrew-cask-versions/pull/9542#issuecomment-685866416>

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/blender.rb [hunk 1]
```diff
@@ -8,6 +10,8 @@
   name "Blender"
   homepage "https://www.blender.org/"
 
+  conflicts_with cask: "blender-lts"
```
Comment (reitermarkus) [inline Casks/blender.rb:13]:
```suggestion
  conflicts_with cask: "homebrew/cask-versions/blender-lts"
```

Comment (walles) [reply Casks/blender.rb]: Done.

**Comment by walles** [comment]
I don't understand the CI failure. Help?

**Comment by miccal** [comment]
> I don't understand the CI failure. Help?

The CI machine is not seeing the `blender-lts` version yet, so nothing to worry about.

File: Casks/blender.rb [hunk 2]
```diff
@@ -8,6 +8,8 @@
   name "Blender"
```
Comment (miccal) [inline Casks/blender.rb:8]: Please add:
```
desc "Free and open-source 3D creation suite"
```
to match `blender-lts`.

Comment (walles) [reply Casks/blender.rb:8]: Done

**Comment by walles** [comment]
How long should one wait for `blender-lts` to become visible before starting to worry?

**Comment by reitermarkus** [comment]
I don't think the `homebrew/cask-versions` tap is updated on CI, so it's not showing up.

**Comment by walles** [comment]
Well, CI passed now, so I think this should be ready for merge.

Thanks for fixing CI @reitermarkus, impressive response time there!

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**